### DISCUSSION TITLE
posts/lin-jiang-xian/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#25] [Announcements]
# posts/lin-jiang-xian/

临江仙

https://blog.wohin.me/posts/lin-jiang-xian/

**Comment by AbuCTF** [comment]
11 years and counting, props to you !

**Reply by brant-ruan** [reply]
Thanks! This has been a joy and habit for me to maintain the blog :)

**Reply by AbuCTF** [reply]
Oh, you're still replying actively :) 
I feel like I'm speaking to someone from the past haha.
